### ISSUE TITLE
[css-text] Make references more specific

### ISSUE THREAD

**Issue by r12a** [w3c/csswg-drafts#5048]
10.2. Character Class Spacing: the text-spacing property
https://www.w3.org/TR/css-text-4/#text-spacing-property

> Note: A commonly used algorithm for determining this behavior is specified in [JLREQ].

We're writing documents in HTML, where we can point to relevant sections in documents, but these references are the equivalent of saying "Here's a book, it's in there somewhere, i'll let you figure out if you can find what i was thinking of." (literally, since i have a copy of the 400-page, published JLREQ book on my shelf.)

It's fine to keep the [JLREQ] footnote reference, but please also point to the appropriate section(s) in jlreq as well.

thank you. rant over.

**Comment by fantasai** [comment]
@r12a Alright, sure, I'll add a section reference. But first, how about you post an updated publication? The current one is from 2012. ;)

**Comment by r12a** [comment]
Well the JLREQ group has been working on it, and i'm lead to believe that they are close to publishing an update. However, it won't contain substantive changes, it will just have a few errata addressed and change the format to a respec-based, multilingual doc.  So i wouldn't have thought you'd need to wait on the publication before adding the references (?)

**Comment by frivoal** [comment]
The correct link is this one, right? https://www.w3.org/TR/jlreq/#appendix_2

**Comment by xfq** [comment]
> Note: A commonly used algorithm for determining this behavior is specified in [JLREQ].

This sentence is current mentioned in the description of [ideograph-alpha](https://www.w3.org/TR/css-text-4/#valdef-text-spacing-ideograph-alpha) and [ideograph-numeric](https://www.w3.org/TR/css-text-4/#valdef-text-spacing-ideograph-numeric). I think they're mentioned in [3.2.6 d of jlreq](https://w3c.github.io/jlreq/#id209):

> Inter-character spacing, between hiragana (cl-15), katakana (cl-16) or ideographic characters (cl-19) and Western characters or European numerals, is quarter em spacing

Appendix B (the link provided by @frivoal) seems to be a superset of it, because it contains a [table](https://w3c.github.io/jlreq/tables/table_en2.pdf), and the amount of spacing between two adjacent characters of all character classes, including ideographs and non-ideographic letters/numerals, are in it.

-----

FWIW, in clreq, it is mentioned in [§ 3.2.2 Mixed Text Composition in Horizontal Writing Mode](https://w3c.github.io/clreq/#mixed_text_composition_in_horizontal_writing_mode):

> In principle, there is tracking or spacing between an adjacent Han character and a Western character of up to one quarter of a Han character width, except at the line start or end.

And [§ 3.2.4 Handling Western Text in Chinese Text Using Proportional Western Fonts](https://w3c.github.io/clreq/#handling_western_text_in_chinese_text_using_proportional_western_fonts):

> Tracking or spacing between a Han character and a Western letter or numeral is up to a quarter of the width of a Han character.

**Comment by frivoal** [comment]
Not sure that https://w3c.github.io/jlreq/#id209 (or the clreq references) adds more than what is already stated in css-text, but Appendix B of JLReq does have more to say.

That said, it would seem more appropriate to use as a note attached to the `auto` value than to the explicit ones.

**Comment by xfq** [comment]
> Not sure that https://w3c.github.io/jlreq/#id209 (or the clreq references) adds more than what is already stated in css-text, but Appendix B of JLReq does have more to say.
> 
> That said, it would seem more appropriate to use as a note attached to the `auto` value than to the explicit ones.

Agreed. And for clreq, it's "**up to** one quarter of a Han character width" (i.e., flexible), instead of the 1/4em spacing as specified in `ideograph-alpha` and `ideograph-numeric`, so it might be more suitable for `auto`.
